sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -206,9 +210,107 @@ public partial class RegistryArtifact
+        public virtual Azure.Response<Azure.Containers.ContainerRegistry.Specialized.UploadManifestResult> UploadManifest(System.IO.Stream manifestStream, Azure.Containers.ContainerRegistry.Specialized.UploadManifestOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Containers.ContainerRegistry.Specialized.UploadManifestResult>> UploadManifestAsync(Azure.Containers.ContainerRegistry.Specialized.OciManifest manifest, Azure.Containers.ContainerRegistry.Specialized.UploadManifestOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Containers.ContainerRegistry.Specialized.UploadManifestResult>> UploadManifestAsync(System.IO.Stream manifestStream, Azure.Containers.ContainerRegistry.Specialized.UploadManifestOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class DownloadBlobResult : System.IDisposable
+    {
+        internal DownloadBlobResult() { }
+        public System.IO.Stream Content { get { throw null; } }
+        public string Digest { get { throw null; } }
+        public void Dispose() { }
+    }
+    public partial class DownloadManifestOptions
+    {
+        public DownloadManifestOptions(string tag = null, string digest = null) { }
+        public string Digest { get { throw null; } }
+        public string Tag { get { throw null; } }
+    }
+    public partial class DownloadManifestResult : System.IDisposable
+    {
+        internal DownloadManifestResult() { }
+        public string Digest { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.Specialized.OciManifest Manifest { get { throw null; } }
+        public System.IO.Stream ManifestStream { get { throw null; } }
+        public void Dispose() { }
+    }
+    public partial class OciAnnotations
+    {
+        public OciAnnotations() { }
+        public System.Collections.Generic.IDictionary<string, object> AdditionalProperties { get { throw null; } }
+        public string Authors { get { throw null; } set { } }
+        public System.DateTimeOffset? CreatedOn { get { throw null; } set { } }
+        public string Description { get { throw null; } set { } }
+        public string Documentation { get { throw null; } set { } }
+        public string Licenses { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        public string Revision { get { throw null; } set { } }
+        public string Source { get { throw null; } set { } }
+        public string Title { get { throw null; } set { } }
+        public string Url { get { throw null; } set { } }
```

Comment:
Should this property be of type Uri, and also named Uri?

Reply:
These properties do represent URLs, so I think this is more appropriate. This is similar to the usage in several SDKs, including in ACR. For example in `Descriptor.Urls` and `Annotations.Url`.